IM_calculation/IM/im_calculation.py

```diff
@@ -413,28 +435,35 @@ def compute_measures_multiprocess(
     ) = constants.Components.get_comps_to_calc_and_store(comp)
 
     bbseries, station_names = get_bbseis(input_path, file_type, station_names)
-
     total_stations = len(station_names)
+    # determine the size of each iteration base on num of processers and mem
     steps = get_steps(
         input_path, process, total_stations, "FAS" in ims and bbseries.nt > 32768
     )
 
     all_results = []
     p = Pool(process)
```

Comment:
This should be moved to just before the while loop and used with a context manager
Claudio will be more than happy to explain why :P